src/steps/check-lead-activity.ts

```diff
@@ -74,6 +97,9 @@ export class CheckLeadActivityStep extends BaseStep implements StepInterface {
         ]);
       }
 
+      const headers = { id: 'ID', leadId: 'Lead ID', activityDate: 'Activity Date', activityTypeId: 'Activity Type ID' };
+      const activityRecords = this.table('matchedActivities', `Matched ${activityType.name} Records`, headers, activities);
```

Comment:
- [ ] After seeing it in context, the name here looks a little confusing.  Perhaps we put the activity name in quotes and change `Records` to `Activities`?

e.g. `Matched "New Lead" Activities`